### ISSUE TITLE
All my AV programs stopped working simultaneously.

### ISSUE THREAD

**Issue by CinderBH** [dragokas/hijackthis#227]
Welcome!
Thank you for joining the section of VIRUSNET association support.
_________________________________________________________________
### BEFORE ASKING HELP, READ CAREFULLY THIS INSTRUCTION:
_________________________________________________________________

### Step 1: Show us the required logs (for PC cure only):

  * Read carefully: "How to make a request for help in the PC cure section":
https://github.com/dragokas/hijackthis/wiki/How-to-make-a-request-for-help-in-the-PC-cure-section%3F

  * Attach 'Collection-[Date].zip' log created by "AutoLogger" tool:
https://www.safezone.cc/resources/autologger-regist-drongo.59/download

### Step 2: Describe your problem in details:

I was just done updating my GPU drivers (official updater app, naturally) when out of nowhere all my AV programs glitched, even though they were working fine (I'd just completed a MalwareBytes Anti-Malware scan hours ago). Windows Security no longer detects Comodo Internet Security even though it's running. Malwarebytes Anti-Malware is stuck as the tray icon with realtime protection turned off, and clicking on the tray icon does nothing. Turning Defender on doesn't work. I even tried downloading Defender from the Microsoft app store and activating it, but that one had realtime protection turned off too.
What should I do? I hadn't exposed myself to any potential virus sources so I really can't imagine what's going on. I'm running Windows 10 Home, for the record.

EDIT: Attaching the log didn't seem to work, so here's an upload: https://drive.proton.me/urls/1FGQMWCKCW#X6klmDXjhLcm

**Comment by dragokas** [comment]
Hi, 
thank you for the log.
We'll return to you as soon as possible.

----------------------------------------------------------------------------------------------------
**Please, note that only members of [VIRUSNET-Association](https://github.com/VIRUSNET-Association) are allowed to respond to PC cure topics. 
Ignore any recommendations given by other users, including PM !!!**

**Assistance is provided free of charge in our free time. If you found our help useful, you can thank us with any amount using [this form](https://safezone.cc/donate/) or you can leave feedback in [Guestbook](https://github.com/dragokas/hijackthis/issues/47).**

**Comment by Sandor-Helper** [comment]
Hi,
Installing several anti-virus programs simultaneously - is not good idea. They can conflict with each other.
I recommend to uninstall them:

> COMODO Internet Security Premium
> COMODO Internet Security Pro
> Malwarebytes version 4.6.7.301

and see what happen. How will the Defender behave after that?

Lets have some more logs:
Please download [Farbar Recovery Scan Tool](https://www.bleepingcomputer.com/download/farbar-recovery-scan-tool/) and save it to your Desktop.

**Note**: You need to run the version compatible with your system. If you are not sure which version applies to your system download both of them and try to run them. Only one of them will run on your system, that will be the right version.

- Right click to run as administrator. When the tool opens click **Yes** to disclaimer.
- Press **Scan** button.
- It will produce logs called **FRST.txt** and **Addition.txt** in the same directory the tool is run from.
- Please attach the logs back here.

Logs you can simply drop to your answer here, see link below "Paste, drop, or click to add files".

**Comment by CinderBH** [comment]
The first two CIS are actually the same program. Also I don't run them all simultaneously, I keep MBAM on the side as a second-opinion scanner.

That being said, sorry about this. I made the mistake of not expecting an expedite reply, so I turned the issue to the MBAM forums, and they also had me run FRST, as well as AdwCleaner. After two rounds of fixing with FRST (well, on the first one actually), both the AVs and Security Center worked correctly again. Though we're not done yet, I'm waiting for a reply on the second round of fixing.
Should I share the logs, still?

**Comment by Sandor-Helper** [comment]
Ok, no, you don't need to share. I see your topic there.
This thread should be closed then.

**Comment by CinderBH** [comment]
Understood.
Could I ask for some advice on some separate issue I've been having (or maybe not so separate after all) or is that outside of your obligations?

**Comment by Sandor-Helper** [comment]
You better ask there at Malwarebytes so as not to confuse the consultant.

**Comment by CinderBH** [comment]
Figure I should. I just had the same doubt about asking as I did here.
Thank you for your patience.

**Comment by dragokas** [comment]
@CinderBH we had day off at that moment.
You may ask here only questions directly related to malware, however, only after completing the treatment on another forum.
For reference, I'll leave a [link](https://forums.malwarebytes.com/topic/305860-all-my-av-programs-stopped-working-simultaneously/)
Good luck!

**Comment by CinderBH** [comment]
(Still, I feel compelled to ask your opinion: IS it an infection? When I thought about it later I felt like the consultant didn't really answer the question, but maybe it's just me.

**Comment by Sandor-Helper** [comment]
In my opinion - all this is not a virus or malware, but system problems.

**Comment by CinderBH** [comment]
Yeah, that makes sense I suppose. I've been careful enough that I haven't got a virus since at least 2009.
I guess I just suspected it because disabling AVs sounds like something a virus would do specifically. I remember one I saw at a net cafè years ago that would force a reboot if you tried to run AdwCleaner.
